pom.xml

```diff
@@ -96,8 +96,8 @@
     <gson-extras.version>0.2.1</gson-extras.version>
     <guava.version>15.0</guava.version>
     <jetty.version>9.2.15.v20160210</jetty.version>
-    <httpcomponents.core.version>4.3.3</httpcomponents.core.version>
-    <httpcomponents.client.version>4.3.6</httpcomponents.client.version>
+    <httpcomponents.core.version>4.4.1</httpcomponents.core.version>
+    <httpcomponents.client.version>4.5.1</httpcomponents.client.version>
```

Comment:
upgrade httpcomponents.client to 4.5.1 otherwise I can not use a custom `RequestConfig`, and this also force me to upgrade httpcomponents.core to 4.4.1, otherwise zeppelin server would fail to start.